src/tests/components/Input.test.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+
+import Input from '../../components/Input';
+import testData from '../fixtures/inputData';
+
+describe('Input', () => {
+  it('should render Input correctly with test data', () => {
+    const wrapper = shallow(<Input {...testData} />);
+    expect(wrapper).toMatchSnapshot();
+  });
+
+  it('should render Input and error message correctly with errorMsg', () => {
+    const wrapper = shallow(<Input {...testData} errorMsg="error message" />);
+    expect(wrapper).toMatchSnapshot();
```

Comment:
better would be finding if error exists instead of snapshots.
I suggest you to read some articles about snapshots. We are using them, but using blindly is leading to many problems ;) I won't tell you more.